rcode/rcode.py

```diff
@@ -60,7 +60,8 @@ def next_open_socket(socks: Sequence[Path]) -> Path:
 
 def is_remote_vscode() -> bool:
     code_repos = Path.home().glob(".vscode-server/bin/*")
-    return len(list(code_repos)) > 0
+    is_ssh = len(os.getenv('SSH_CLIENT', '')) > 0
+    return len(list(code_repos)) > 0 and is_ssh
```

Comment:
is_ssh = os.getenv('SSH_CLIENT') is Fine I think? if no `SSH_CLIENT` return None.

```python3
return  len(list(code_repos)) > 0 and os.getenv('SSH_CLIENT')
```